src/tests/orca-jedi/test_geometry.cc

```diff
@@ -32,7 +32,7 @@ CASE("test basic geometry") {
   nemo_var_mappings[1].set("name", "sea_ice_area_fraction_error")
     .set("nemo field name", "sic_tot_var")
     .set("model space", "surface")
-    .set("variable type", "background standard deviation");
+    .set("variable type", "background error standard deviation");
```

Comment:
Should this be variance to match the nemo field name?

Reply:
Sorry about these oversights! I have made the changes in the latest commit. Hopefully that covers all of them.